precompiles/Playground.yul

```diff
@@ -623,6 +628,180 @@ object "Playground" {
                 c100, c101, c110, c111, c120, c121 := fp6Sub(z00, z01, z10, z11, z20, z21, c100, c101, c110, c111, c120, c121)
             }
 
+            ////////////////////////////////////////////////////////////////
+            //                      FROBENIUS
+            ////////////////////////////////////////////////////////////////
```

Comment:
We should start using a single comment to separate the different sections. Something like `// FROBENIUS`